src/Mint/Common.hs

```diff
@@ -342,10 +326,12 @@ pRemove common vrange discConfig outs sigs = plam $ \pkToRemove node -> P.do
                         # out
                         #== configF.penaltyAddress
                         #&& ownInputFee
-                        #<= plovelaceValueOf
+                        #<= pvalueOf
                         # (pfield @"value" # out)
+                        # configF.stakeCS
+                        # configF.stakeTN
```

Comment:
This might be more readable:
```haskell
    (pany # plam (\out -> pfield @"address #out #== configF.penaltyAddress
                          #&& ownInputFee #<= pvalueOf # configF.stakeCS # configF.stakeCS)
          # outs)
```

or something similar that the formatter allows.

Maybe it could even be:

```haskell
  let penaltyFeePaid = plam $ \out -> ...
  let finalCheck =
    ...
    (pany # penaltyFeePaid # outs)
  ...
```